examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -57,7 +39,7 @@ def plot_matrices(matrices, matrix_kind):
 
 masker = input_data.NiftiMapsMasker(
     msdl_data.maps, resampling_target="data", t_r=2, detrend=True,
-    low_pass=.1, high_pass=.01, memory='nilearn_cache', memory_level=1)
+    low_pass=.1, high_pass=.01, memory='nilearn_cache', memory_level=1).fit([])
```

Comment:
What's the reasoning for adding the `.fit()` here distinct from `.fit_transform()` as previously ? It feels less explicit, as a reader.

Reply:
because the masker is not fitted on the data provided to transform (or fit_transform)